evap/contributor/templates/contributor_course_form.html

```diff
@@ -92,6 +94,20 @@
     </form>
 {% endblock %}
 
+{% block modals %}
+    {{ block.super }}
+    {% trans "Approve course" as title %}
+    {% trans "If you approve this course you won't be able to make any additional changes afterwards, unless the student representatives reopen the course for changes.<br /><strong>Please approve the course when you're done editing it.</strong>" as question %}
```

Comment:
reopening: yes, that's kind of what i want. otherwise some editors will fear to make irreversible changes when approving an therefore hesitate to do that. but maybe we can just ignore that.